Sources/SweetCollectionViewController.swift

```diff
@@ -17,8 +15,6 @@ public class SweetCollectionViewController: UIViewController {
         self.init()
         self.collectionView = UICollectionView(frame: .zero, collectionViewLayout: collectionViewLayout)
         self.collectionView.translatesAutoresizingMaskIntoConstraints = false
-        self.collectionView.delegate = self
-        self.collectionView.dataSource = self
```

Comment:
No, this one replaces the lazy-loaded collection view so you can override its layout.